docs/input/docs/installation/configure.md

```diff
@@ -0,0 +1,52 @@
+---
+Order: 20
+Description: Configure BBT.StructureTools
+---
+
+# General
+
+The BBT Structure Tools are entirely configurable by code. This section covers the
+correct IoC container registration of the infrastructure components.
+
+## Preconditions
+
+* An IoC container can be used to resolve registrations as well as internally by the library.
+
+## IoC Container
+
+The library is usable with any IoC container, but it must support some preconditions:
+
+* Injection to non-public constructors
+* Registration of unbound generic types
+
+### Initialization
+
+Before the registrations can be done the `IocHandler` resolver must be set.
+Implement the resolver using the `IIocResolver`.
+Once this is done you can configure the `IocHandler`:
+
+```csharp
+var resolver = new YourResolver(); // Your implementation of the IIocResolver
+IocHandler.Instance.IocResolver = resolver;
+```
+
+Afterwards make sure that the dependencies from the [BBT.StrategyPattern] are registered.
+See [BBT.StrategyPattern usage with IoC] for instructions.
+
+In order for the library to work correctly it needs to register some types into the IoC container
+itself. As the implementations are private to allow for a concise API it needs to do this by itself within
```

Comment:
"to allow for a concise" -> to allow a concise

Reply:
Thanks, adjusted